MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,447 @@
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:DataFormViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding DataModel}"/>    
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.DataObject = new DataModel();
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Defining editors
+
+The data form control automatically generates `DataFormItems` (which has UI settings of data field) based on the data type in the `SfDataForm.DataObject` property. The `DataFormItem` encapsulates the layout of label and editor setting for the data field appearing in the dataform. 
+
+The type of input editor generated for the data field depends on the type and attribute settings of the property. The following table lists the `DataFormItem` and its constraints for generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Data Type / Attribute</th>
+</tr>
+<tr>
+<td>
+{{'[DataFormTextItem]
+`Link`
+</td>
+<td>
+Default DataFormItem generated for the String type and the properties with [DataType(DataType.Text)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormMultiLineTextItem]
+`Link`
+</td>
+<td>
+Generated for string type property with [DataType(DataType.MultilineText)] attributes.
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormPasswordTextItem]
+`Link`
+</td>
+<td>
+Generated for string type property with [DataType(DataType.Password)] attributes.
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormCheckBoxItem]`Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormSwitchItem] `Link`
+</td>
+<td>
+Generated for the Bool type property.
+[BoolDataTypeAttribute]
+</td>
+</tr>
+
+<tr>
+<td>
+{{'[DataFormDateItem] `Link`
+</td>
+<td>
+Generated for the DateTime, DateOnly, DateTimeOffset type property.
+[DataType(DataType.Date)].
+[DataType(DataType.DateTime)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormTimeItem]
+`Link`
+</td>
+<td>
+Generated for the TimeSpan, TimeOnly  type property.
+[DataType(DataType.Time)].
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormPickerItem] `Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+
+<tr>
+<td>
+{{'[DataFormAutoCompleteItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormComboBoxItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+<tr>
+<td>
+{{'[DataFormRadioGroupItem]`Link`
+</td>
+<td>
+Generated for the Enum type property.
+[EnumDataTypeAttribute]
+</td>
+</tr>
+</table>
+
+The following list of editors are supported:
```

Comment:
As stated above after inluded the details with above table remove this table.

Reply:
Supported editors table removed